canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.jsx

```diff
@@ -80,21 +85,49 @@ class CheckboxsetControl extends React.Component {
 		}
 		const checkboxes = [];
 		for (var i = 0; i < this.props.control.values.length; i++) {
+			const tooltipId = uuid4() + "-tooltip-" + this.props.control.name;
+			let tooltip = "";
+			if (this.props.control.description && !(this.props.state === STATES.DISABLED || this.props.state === STATES.HIDDEN) && !this.props.tableControl) {
+				tooltip = (
+					<span >{this.props.control.valueDescs[i]}</span>
```

Comment:
How do you know there is a description?  Just because the control has a description doesn't mean the enum values do.

Reply:
I just Updated the paramDef resources, they are being fetched from the associated JSON files.

Reply:
There won't be a description at the value level in all cases so we wouldn't want tooltips showing up in that case.

Reply:
Showing only tooltips with desc